app/tests/unit/components/Dashboard.test.tsx

```diff
@@ -115,4 +118,58 @@ describe("The Dashboard", () => {
       "/cif/project-revision/mock-id-1/form/0?anchor="
     );
   });
+  it("Shows dashboard links with admin privileges", () => {
```

Comment:
Should there also be a test to confirm that non-admins can't see the admin info?